src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -502,7 +501,7 @@ export class StudyViewPageStore {
                     const entityIdKey = clinicalAttribute.patientAttribute
                         ? 'patientId'
                         : 'sampleId';
-                    let data = await client.fetchClinicalDataUsingPOST({
+                    let data = await defaultClient.fetchClinicalDataUsingPOST({
```

Comment:
what's the idea of the default client?  it can be switched based on condition?

Reply:
its just a cleanup `defaultClient` and `client` points to same instance of `shared/api/cbioportalClientInstance`